src/ext.js

```diff
@@ -105,114 +105,139 @@ const getStyleSettings = (env) => {
       type: 'items',
       items: [
         {
-          component: 'style-editor',
+          component: 'styling-panel',
+          chartTitle: 'Object.StraightTable',
```

Comment:
This might be changed in the future, since the customer-facing name for sn-table is not decided yet. But this is fine for now

Reply:
Yes sure